osu.Game.Rulesets.Taiko/UI/TaikoPlayfield.cs

```diff
@@ -255,16 +267,30 @@ internal void OnNewResult(DrawableHitObject judgedObject, JudgementResult result
 
                     break;
             }
-        }
 
-        private class ProxyContainer : LifetimeManagementContainer
-        {
-            public new MarginPadding Padding
+            if (characterDrawable.Drawable is DrawableTaikoMascot mascot)
             {
-                set => base.Padding = value;
+                var isFailing = result.Type == HitResult.Miss;
+
+                // Only take combo in consideration when it's not a strong hit (it's always false)
```

Comment:
what's always false? I don't understand what this interjection is trying to say.

Reply:
peppy explained to me that strong hits have their AffectsCombo return `false` because they need multiple hits (IIRC), that's why I'm exempting them to the next check.

Reply:
If this has to do with `AffectsCombo`, why not check that directly? It's available through `JudgementResult.Judgement`, and avoids the need to keep track whether that's true or false for particular hitobject types.

Reply:
Aren't I'm already using this? Thinking about it I might just remove it but I also know there must be a reason why I included it in the first place.

Reply:
I figured out why I added it: strong hits that are only played with only one key have their JudgementResult type set to Miss.